packages/slate-react/src/components/editable.tsx

```diff
@@ -1722,9 +1783,9 @@ export type RenderPlaceholderProps = {
  */
 
 export const DefaultPlaceholder = ({
-  attributes,
-  children,
-}: RenderPlaceholderProps) => (
+                                     attributes,
```

Comment:
not sure this linting change was intentional?

Reply:
Sorry, it wasn't intentional. Corrected here -> [aded41d](https://github.com/ianstormtaylor/slate/pull/5648/commits/aded41dde434fb838c1ecbf0d5dd55c36b810299)